heidrun/ct_dc.rb

```diff
@@ -0,0 +1,95 @@
+
+# If more than one dc:contributor property exists then it drops the last one
+# If one or zero exist return nil
```

Comment:
@moltude is there anything wrong with returning `nil` here and doing `map(...).flatten.compact` below?